tgui/packages/tgui/interfaces/SlotMachine.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright (c) 2020 @ZeWaka
+ * SPDX-License-Identifier: ISC
+ */
+
+import { useBackend } from '../backend';
+import { Box, Button, NoticeBox, Divider, BlockQuote, Icon } from '../components';
+import { Window } from '../layouts';
+
+export const SlotMachine = context => {
```

Comment:
Double check this works. I know I said to remove unneeded arguments, but I think `context` is the second argument passed in?